lib/linguist/languages.yml

```diff
@@ -2527,8 +2527,7 @@ Ignore List:
   group: INI
   aliases:
   - ignore
-  - gitignore
-  - git-ignore
+  - git ignore
```

Comment:
Again, could we please keep this PR focussed? This sort of clean-up belongs in a separate submission.